File: Java/백준/Bronze/6825. Body Mass Index/Body Mass Index.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        float a = in.nextFloat();
        float b = in.nextFloat();
        float bmi = a/(b*b);
        if(bmi<18.5){
            System.out.println("Underweight");
        }
        else if(bmi>25){
            System.out.println("Overweight");
        }
        else{
            System.out.println("Normal weight");
        }
    }
}

File: C#/백준/Bronze/6825. Body Mass Index/Body Mass Index.cs
using System;

namespace q{
    class w{
        static void Main(){
            float a = float.Parse(Console.ReadLine());
            float b = float.Parse(Console.ReadLine());
            float bmi = a/(b*b);
            if(bmi<18.5){
                Console.WriteLine("Underweight");
            }
            else if(bmi>25){
                Console.WriteLine("Overweight");
            }
            else{
                Console.WriteLine("Normal weight");
            }
        }
    }
}